FrameworkSystem/DB/ProxyDB.py

```diff
@@ -3,21 +3,24 @@
 
 __RCSID__ = "$Id$"
 
+import os
```

Comment:
Changes to this DB should be validated through an integration test that should sit in /tests/Integration/Framework (to be created)